src/main/java/dk/cngroup/lentils/entity/Cypher.java

```diff
@@ -188,6 +196,20 @@ public void setPlaceDescription(final String placeDescription) {
         this.placeDescription = placeDescription;
     }
 
+    public List<User> getOrganizers() {
+        return organizers;
+    }
+
+    public List<User> getSortedOrganizers() {
```

Comment:
Mám takový pocit, že by měl jít dát OrderBy přímo jako anotace na ten sloupec. A nemusíš mít 2 metody.
jestli si nenarazil.

Reply:
zkusím, dekuju :)

Reply:
Jestli to teda dává smysl.